.isort.cfg

```diff
@@ -2,4 +2,4 @@
 line_length = 79
 multi_line_output = 3
 include_trailing_comma = True
-known_third_party =decouple,dj_database_url,django,drf_yasg,faker,hashid_field,rest_framework,rest_framework_simplejwt
+known_third_party =PIL,cloudinary,decouple,dj_database_url,django,drf_yasg,faker,hashid_field,rest_framework,rest_framework_simplejwt
```

Comment:
remove this from pre-cmmit.yaml file 

-   repo: https://github.com/asottile/seed-isort-config
    rev: v1.9.3
    hooks:
    -   id: seed-isort-config
    -

Reply:
I don't understand here.